developer-extension/src/panel/components/json.tsx

```diff
@@ -275,6 +279,15 @@ function JsonText({
   )
 }
 
+function getSearchQuery(descriptor: JsonValueDescriptor) {
+  const { columnPath } = useContext(JsonContext)!
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏ Does this works? I don't think you can use hooks in a functions like this.

Instead, you could either pass `columnPath` as a function argument or transform this function into a custom hook

Reply:
It works, indeed not good practice. I removed this and passed it in as an argument.